Probesys/bileto
# imp: Rename "operational" to "agent"

## Related issue(s)



https://github.com/Probesys/bileto/issues/576

## How to test manually



1. Navigate to the "roles" and "user authorizations" section
2. Check that all reference to "operational roles" have been replaced by "agent roles"

## Checklist
